utility/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="org.oppia.util">
-  <uses-sdk
+  <uses-sdk android:minSdkVersion="19"
     android:targetSdkVersion="29" />
```

Comment:
@BenHenning Strangely enough, I was able to add back the minSDK without Gradle freaking out. I think the error I was getting before may have been due to the weird Gradle/Android Studio issues I was having earlier in the week!

Reply:
Ah, makes sense. Thanks for re-adding ti!